packages/mask/src/plugins/ITO/SNSAdaptor/ClaimAllDialog.tsx

```diff
@@ -366,7 +368,14 @@ export function ClaimAllDialog(props: ClaimAllDialogProps) {
                             chainId={chainId}
                             setChainId={setChainId}
                             classes={classes}
-                            chains={[ChainId.Mainnet, ChainId.BSC, ChainId.Matic, ChainId.Arbitrum, ChainId.xDai]}
+                            chains={[
+                                ChainId.Mainnet,
+                                ChainId.BSC,
+                                ChainId.Matic,
+                                ChainId.Arbitrum,
+                                ChainId.xDai,
+                                ChainId.Celo,
```

Comment:
I think the ITO plugin now is not working on CELO.

Reply:
I'm not quite sure how it works, how would I test it?

Reply:
@guanbinrui would the relevant contracts need to be deployed on Celo or how else can I handle this?

Reply:
Hi again @guanbinrui, sorry to bother you again. Just looking to have more progress on this.

Reply:
Hi again @guanbinrui, I will try deploying with my own address to see how that goes. :)

Reply:
It appears that it would be best to open a PR at https://github.com/DimensionDev/InitialTwitterOffering for Celo. What do you think? @guanbinrui